packages/aws-library/src/aws_library/ec2/client.py

```diff
@@ -200,7 +200,11 @@ async def start_aws_instance(
                     launch_time=instance["LaunchTime"],
                     id=instance["InstanceId"],
                     aws_private_dns=instance["PrivateDnsName"],
-                    aws_public_ip=instance.get("PublicIpAddress", None),
+                    aws_public_ip=(
```

Comment:
Why this change?  `instance.get("PublicIpAddress", None)` is doing exactly the same ... more effective (one call) and shorter :-)

Reply:
OK. it is a revert. All good